L103-python-add-typing-to-sync-api.md

```diff
@@ -19,26 +19,19 @@ This proposal plans to add type annotation to python Sync public APIs which allo
 
 
 ### Related Proposals: 
-* Loosely related: https://github.com/lidizheng/proposal/blob/grpc-python-async-api/L58-python-async-api.md#introduce-typing-to-generated-code
+* Loosely related to [Async API proposal](https://github.com/lidizheng/proposal/blob/grpc-python-async-api/L58-python-async-api.md#introduce-typing-to-generated-code).
   * The proposal includes the part to introduce typing to gRPC Python Async APIs.
 
 
 ## Proposal
 
-Add type annotation to the following Python APIs:
-* All APIs in `__init__.py` (the actuall list is too long, please refer to following PR for changes):
+Add type annotation to the following:
+* All public Python APIs listed in [`__all__` variable](https://github.com/grpc/grpc/blob/54dd7563c2d563bff74e4b558f2e985db4a01f2d/src/python/grpcio/grpc/__init__.py#L2100) (the actuall list is too long, please refer to the following PR for changes):
+  * [PR link holder]
+* gRPC Python generated Code.
+  * [PR Link holder]
+* Python ancillary packages.
```

Comment:
You should probably list out the ancillary packages explicitly. This isn't really a well known term. These are:
- `grpcio-admin`
- `grpcio-channelz`
- `grpcio-csds`
- `grpcio-health-checking`
- `grpcio-reflection`
- `grpcio-status`
- `grpcio-testing`